EndlessClient/Rendering/Character/CharacterAnimationActions.cs

```diff
@@ -422,7 +422,7 @@ public interface ICharacterAnimationActions
     {
         void Face(EODirection direction);
 
-        void StartWalking(Option<MapCoordinate> targetCoordinate);
+        void StartWalking(Option<MapCoordinate> targetCoordinate, bool ghosted);
```

Comment:
I think a default value of `=false` makes sense both here and in `CharacterActions`.